packages/eyes/src/generate-snapshots/filter-logic.ts

```diff
@@ -0,0 +1,32 @@
+export function generateFilteringLogic(mapping: any): (stylableModule: any) => any {
```

Comment:
It would be great to have a comment here explaining what this is all about. Maybe some of the any's could be replaced with types as well?